src/pkg/packaging/deb/dotnet-runtime-deps-debian_config.json

```diff
@@ -0,0 +1,49 @@
+{
+    "maintainer_name":"Microsoft",
+    "maintainer_email": "dotnetcore@microsoft.com",
+
+    "package_name": "%RUNTIME_DEPS_DEBIAN_PACKAGE_NAME%",
+    "install_root": "/usr/share/dotnet",
```

Comment:
Is this the best place for this?  Do we technically install anything here?

Reply:
install_root is a required field for package schema. No, no payload is installed in this location.